brainstormforce/astra
# AST-1456 - Jerk issue while switching to dark mode site

### Description
- First it loads color palette 1 then switching to dark palette there is jerk.

### Screenshots
- https://share.bsf.io/GGu49mYg

### Types of changes
- Bug fix 

### How has this been tested?
- Enable dark OS setting
- Choose OS Aware setting for first load
- Then check in incognito mode, jerk will be there

### Checklist:
